src/unimport/utils.py

```diff
@@ -76,12 +74,19 @@ def is_std(package: str) -> bool:
 
 @functools.lru_cache(maxsize=3)
 def action_to_bool(action: str) -> bool:
-    if action == "":
```

Comment:
I believe this introduced a regression. Before, when `action == ""`, the function would return `True`. Now it will raise.